test/Microsoft.ML.Benchmarks/TrainPredictionBench.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class TrainPredictionBench
+    {
+        internal static ClassificationMetrics s_metrics;
+        static PredictionModel<IrisData, IrisPrediction> s_trainedModel;
+
+        [GlobalCleanup]
+        public void Accuracy()
+        {
+            var dataPath = Program.GetDataPath("iris.txt");
+            var testData = new TextLoader<IrisData>(dataPath, useHeader: true, separator: "tab");
+            var evaluator = new ClassificationEvaluator();
+            s_metrics = evaluator.Evaluate(s_trainedModel, testData);
+        }
+
+        [Benchmark]
+        public void Iris()
+        {
+            var dataPath = Program.GetDataPath("iris.txt");
+            var pipeline = new LearningPipeline();
+
+            pipeline.Add(new TextLoader<IrisData>(dataPath, useHeader: true, separator: "tab"));
+            pipeline.Add(new ColumnConcatenator(outputColumn: "Features",
+                "SepalLength", "SepalWidth", "PetalLength", "PetalWidth"));
+
+            pipeline.Add(new StochasticDualCoordinateAscentClassifier());
+
+            PredictionModel<IrisData, IrisPrediction> model = pipeline.Train<IrisData, IrisPrediction>();
+
+            IrisPrediction prediction = model.Predict(new IrisData()
```

Comment:
>Predict [](start = 46, length = 7)

Best to separate into two separate tests: 1. for training another for interence/predictions.
Those two are measured independently and are tuned independently

Reply:
Good point. I split training and prediction into two separate tests